scss/_variables.scss

```diff
@@ -971,6 +976,11 @@ $input-line-height-lg:                  $h5-line-height !default; // Boosted mod
 $input-transition:                      border-color $transition-duration $transition-timing, $transition-focus !default;
 
 $form-color-width:                      3rem !default;
+
+// Boosted mod: Quantity selector specific variables
+$quantity-selector-l-width:             7.5em !default;
```

Comment:
Why is it separated from its friends? (lines 1850-1864)

Reply:
Done